content/sensu-enterprise/2.8/guides/snmp-sensu-guide.md

```diff
@@ -0,0 +1,252 @@
+---
+title: "How to Monitor Devices With Sensu and SNMP"
+product: "Sensu Enterprise"
+version: "2.8"
+weight: 1
+menu:
+ sensu-enterprise-2.8:
+   parent: guides
+---
+
+# Objectives
```

Comment:
Might want to add "adding mibs"

Reply:
For the purpose of the guide, I don't think that adding MIBs should be an objective. In this case, the guide's purpose is to show how the extension works by way of installation, a basic configuration, and testing with the `snmptrap` command.